client/src/store/reducers.js

```diff
@@ -60,11 +63,15 @@ function listsApp(state = initialState, action) {
 			});
 
 		case DELETE_LIST:
-			return Object.assign({}, state, {
-				lists: state.lists.filter((list) => {
-					return list.id !== action.id;
-				}),
-			});
+			return {
+				...state,
+				lists: state.lists.filter((list) => list.id !== action.id)
+			}
+			// return Object.assign({}, state, {
+			// 	lists: state.lists.filter((list) => {
+			// 		return list.id !== action.id;
+			// 	}),
+			// });
```

Comment:
Just another way of doing the same thing. It's mostly personal preference if you use `...spread` or `Object.assign` 👍